src/analyzer.ts

```diff
@@ -98,6 +98,10 @@ export class Analyzer {
     return this._context.load(resolvedUrl, providedContents);
   }
 
+  canResolveUrl(url: string): string {
```

Comment:
Derp about `string -> boolean`.  Fixed.

The use case is that `polymer-build` will use this to inquire about whether a dependency is external to the package or not.

Added test to express the expectation of this API.